src/components/BodyRouter.js

```diff
@@ -4,10 +4,14 @@ import ClimbAttemptForm from './ClimbAttemptForm';
 import AppContext from './AppContext';
 import History from './History';
 import { useContext } from 'react';
+import Paper from '@mui/material/Paper';
+import Button from '@mui/material/Button';
+import Stack from '@mui/material/Stack';
 
 function BodyRouter(props) {
     let [context, setContext] = useContext(AppContext);
     return <>
+        <Paper elevation={3}>
         <div id='pageBody'>
```

Comment:
Nit: indentation

Reply:
Also, if this div exists only to wrap the same thing that the Paper component wraps just combine the two.

Reply:
Removed pageBody class and added to parent div.